File: Back End/Python/Chapter 5 - Control Statements/2K221229/patteren_star_pyramid_ascending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Star pyramid Pattern - Ascending
#
# ####*
# ###* *
# ##* * *
# #* * * *
# * * * * *
#
# Here, <blank-space> is denoted as #

limit = int(input("Enter the limit: "))

for i in range(1, limit + 1, 1):
    for j in range(i, limit + 1, 1):
        print(" ", end='')
    for j in range(1, i + 1, 1):
        print("* ", end='')
    print(end='\n')

File: Software/C/Chapter 3 - Control Statement/2K221008/patteren_star_pyramid_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Star pyramid Pattern - Ascending

####*
###* *
##* * *
#* * * *
* * * * *

Here, <blankspace> is denoted as #

Algorithm:--->



*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1;i<=count;i++){
        for(j=i;j<count;j++){
            printf(" ");
        }
        for(j=1;j<=i;j++){
            printf("* ");
        }
        printf("\n");
    }
}
